components/toasts.css

```diff
@@ -0,0 +1,49 @@
+.gentle-snackbar-popup {
+  position: fixed;
+  min-width: 15rem;
+  background-color: var(--dark-indicator);
+  padding: 1.2rem;
+  border-radius: 10px;
+  display: flex;
+  gap: 1rem;
+  z-index: 2;
+  box-shadow: 4px 4px #212121;
+}
+.gentle-snackbar-text {
+  color: var(--text-color);
+  font-family: "Roboto-slab", sans-serif;
+}
+.snackbar-actions {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+.gentle-snackbar-link {
+  text-decoration: none;
+  color: #af8ddd;
+  border-radius: 40px;
```

Comment:
would suggest rems for size greater than 10px

Reply:
Done [b7eeec6](https://github.com/dev-enforced/Gentle_UI/pull/15/commits/b7eeec6ad460d8060f7c1dc50eb7495926fb5647)